src/components/views/PCI.js

```diff
@@ -0,0 +1,169 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+
+import {
+  AccordionSet,
+  Button,
+  Col,
+  ExpandAllButton,
+  Headline,
+  Layout,
+  Pane,
+  PaneMenu,
+  Row,
+  Spinner,
+} from '@folio/stripes/components';
+
+import { IfPermission } from '@folio/stripes/core';
+
+import {
+  Agreements,
+  AgreementsList,
+  PCICoverage,
+  PCIInfo,
+  ParentPackageDetails,
+  TitleInfo,
+} from '../EResourceSections';
+
+export default class PCI extends React.Component {
+  static propTypes = {
+    data: PropTypes.shape({
+      eresource: PropTypes.shape({
+        id: PropTypes.string,
+        name: PropTypes.string,
+        status: PropTypes.shape({
+          label: PropTypes.string,
+        }),
+      }).isRequired,
+      relatedEntitlements: PropTypes.array,
+    }),
+    handlers: PropTypes.shape({
+      onClose: PropTypes.func.isRequired,
+    }),
+    isLoading: PropTypes.bool,
+  }
+
+  state = {
```

Comment:
Let's not write new views with the old AccordionSet model that relies on us writing the section togglers. [Use `AccordionStatus` instead.](https://github.com/folio-org/stripes-components/blob/24665cc3e5e7e56811ea31466cb3c3d717bb4c73/lib/Accordion/readme.md#L42)